static/app/components/searchQueryBuilder/tokens/freeText.tsx

```diff
@@ -297,6 +310,15 @@ function SearchQueryBuilderInputInternal({
     currentInputValueRef.current = inputValue;
   }, [inputValue, currentInputValueRef]);
 
+  // Track the usage of ctrl+a so that we can ignore replacing the raw search key and keep
+  // it as a free text token, so that the functionality of doing ctrl+a to select the
+  // text to replace or delete still works as expected.
+  const ctrlABeingPressed = useRef(false);
```

Comment:
I don't think you need to track ctrl+a being pressed? There are other ways to select text like clicking and dragging and shift+arrow. If this is necessary, it's probably best to check the input directly (like `input.selectionStart`)